src/content/docs/pt-br/basics/astro-components.mdx

```diff
@@ -249,7 +249,7 @@ import Involucro from '../components/Involucro.astro';
 ```
 
 :::tip
-Use um atributo `slot="meu-slot"` no elemento filho que você quer passar através de um correspondente placeholder `<slot name="meu-slot"/>` no seu componente.
+Use um atributo `slot="meu-slot"` no elemento filho que você quer passar para um placeholder `<slot name="meu-slot"/>` correspondente no seu componente.
```

Comment:
"Passar para" is more natural here than "passar através"; the latter would make sense if we were passing it through the slot to a third place, but in this case, we're passing the element with the `slot` attribute to the `slot` element.